scribus/plugins/tools/hunspellcheck/hunspellpluginimpl.cpp

```diff
@@ -181,7 +181,7 @@ bool HunspellPluginImpl::parseTextFrame(StoryText *iText)
 			wf.replacements = hspellerMap[wordLang]->suggest(word);
 			wordsToCorrect.append(wf);
 		}
-		currPos = iText->nextWord(wordStart);
+		currPos = iText->endOfWord(wordStart);
```

Comment:
But then the new `nextWord()` is wrong, it should match exactly the behaviour of the old one.